<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LauncherClass</class>
 <widget class="QMainWindow" name="LauncherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>401</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>711</width>
    <height>401</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>711</width>
    <height>401</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Link">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Link">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Link">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>忘却的战场-测试版</string>
  </property>
  <property name="windowIcon">
   <iconset resource="launcher.qrc">
    <normaloff>:/Launcher/Resources/Demo.png</normaloff>:/Launcher/Resources/Demo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>181</width>
      <height>325</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCommandLinkButton" name="commandLinkButton">
       <property name="styleSheet">
        <string notr="true">QCommandLinkButton{
	font: 75 12pt &quot;微软雅黑&quot;;
	
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>窗口模式运行游戏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="commandLinkButton_2">
       <property name="styleSheet">
        <string notr="true">QCommandLinkButton{
	font: 75 12pt &quot;微软雅黑&quot;;
	
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>全屏模式运行游戏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="commandLinkButton_3">
       <property name="styleSheet">
        <string notr="true">QCommandLinkButton{
	font: 75 12pt &quot;微软雅黑&quot;;
	
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>游戏手册</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="commandLinkButton_4">
       <property name="styleSheet">
        <string notr="true">QCommandLinkButton{
	font: 75 12pt &quot;微软雅黑&quot;;
	
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>自述文档</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="commandLinkButton_7">
       <property name="styleSheet">
        <string notr="true">QCommandLinkButton{
	font: 75 12pt &quot;微软雅黑&quot;;
	
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>意见反馈</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="commandLinkButton_5">
       <property name="styleSheet">
        <string notr="true">QCommandLinkButton{
	font: 75 12pt &quot;微软雅黑&quot;;
	
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>安装Directx9.0C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="commandLinkButton_6">
       <property name="styleSheet">
        <string notr="true">QCommandLinkButton{
	font: 75 12pt &quot;微软雅黑&quot;;
	
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>幻爵官方网站</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#ff0000;&quot;&gt;注意!&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ff0000;&quot;&gt;如果在进入战斗界面时程序崩溃,&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ff0000;&quot;&gt;请点击右侧按钮,安装Directx9.0C.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="launcher.qrc"/>
 </resources>
 <connections/>
</ui>
